keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/resource/ImageHandler.kt

```diff
@@ -62,9 +62,7 @@ class ImageHandler(
 
   override suspend fun current(resource: Resource<ImageSpec>): Image? =
     with(resource) {
-      val artifact = DeliveryArtifact(spec.artifactName, DEB)
-      val latestVersion = artifact.findLatestVersion()
-      imageService.getLatestImage(spec.artifactName, latestVersion, "test")
+      imageService.getLatestImage(spec.artifactName, "test")
```

Comment:
Simpler call to just get latest AMI for the app rather than based on the latest artifact version